flume/conf/cygnus_instance.conf.template

```diff
@@ -38,3 +38,6 @@ LOGFILE_NAME=cygnus.log
 
 # Administration port. Must be unique per instance
 ADMIN_PORT=8081
+
+# Polling interval for the configuration reloading
```

Comment:
Fixed in commit https://github.com/telefonicaid/fiware-connectors/commit/bf7dd2244880d3915955fbd5e9e7857483d26cfc